vscode-lean4/package.json

```diff
@@ -379,7 +379,7 @@
 		],
 		"configurationDefaults": {
 			"[lean4]": {
-                                "editor.unicodeHighlight.ambiguousCharacters": false,
+                "editor.unicodeHighlight.ambiguousCharacters": false,
```

Comment:
spurious edit?

Reply:
yes, will undo, but I'm also surprised to see a text file that is using actual tab characters for indentation.  My editor automatically converts those to spaces.